### ISSUE TITLE
Opportunity not being created on new Interaction

### ISSUE THREAD

**Issue by mrchev** [SalesforceFoundation/Interactions-for-Student-Recruitment#14]
I'm testing the Interaction data entry and it is not creating an opportunity, getting error: Audit Reason
Reason: Interaction has an Opportunity Key, but could not find associated Opportunity in the system. Please verify that the Opportunity Key is valid.

I changed the Opportunity Stage values and see on the APEX trigger that "Prospecting" is a stage value listed there, is that my issue?  I was going to play with this in sandbox (just deployed to prod and moving to SB tomorrow) and was wondering if this is a known issue.

**Comment by mrchev** [comment]
The stage are matching, I had removed all the default values and added the stages listed in the documentation during setup. 

Here is the full list of Opportunity Stages in Interaction table:

Inquired
Confirmed Interest
Applied
Evaluated
Admitted
Deposit Received
Enrolled
No Longer Interested
Withdrawn
Denied

This matches the Opportunity Stage list.

**Comment by uwreeser** [comment]
Maybe you found the answer already but the answer could be in the error message you're seeing. Basically interactions was able to create a key, but it errors on the Opportunity creation step. Out of the box, some of the keys depend on Recruitment Interest or Academic Interest being populated, if those aren't then the process fails. Career is also a component of this if memory serves potentially as well.

**Comment by mrchev** [comment]
Hi Scott.

I thought all those were populated, maybe career wasn't.  Anyway, pulled
the plug on the trial since it was failing out of the gate.  If another
chance comes along to try it I'll see what it does, or maybe drop it into a
dev org if I have the time to set it all up.

Thanks for your reply.

Mike

**Comment by uwreeser** [comment]
Yeah and I may be speaking out of turn possibly but I remember running into similar problems and I had to delve into what fields I was setting on the entry of an Interaction record in order to get the result I expected.

**Comment by mrchev** [comment]
I spent hours looking over everything, finally gave up after combing over
logs to see what was going on and still finding no answer.  Wish I had a
sandbox with it installed, would be nice to figure out what was going on so
it can avoided in the future.
